packages/samples/headless-commerce-react/e2e/home.spec.ts

```diff
@@ -0,0 +1,32 @@
+import {test, expect} from '@playwright/test';
+
+test.describe('Home Page', () => {
+  test('should load and display the search box', async ({page}) => {
+    await page.goto('/');
+    const searchBox = page.getByLabel('Enter query');
+    await expect(searchBox).toBeVisible();
+  });
+
+  test('should perform a search and display products', async ({page}) => {
+    await page.goto('/');
+    const searchBox = page.getByLabel('Enter query');
+    await searchBox.fill('blue pants');
```

Comment:
I wonder if we should even enter a query. We could just submit an empty query, no?